fuzzers/qemu/qemu_systemmode/src/fuzzer_classic.rs

```diff
@@ -87,15 +87,28 @@ pub fn fuzz() {
     println!("Breakpoint address = {breakpoint:#x}");
 
     let mut run_client = |state: Option<_>, mut mgr, _core_id| {
+        let target_dir = env::var("TARGET_DIR").expect("TARGET_DIR env not set");
         // Initialize QEMU
-        let args: Vec<String> = env::args().collect();
-        let env: Vec<(String, String)> = env::vars().collect();
+        let qemu = Qemu::builder()
+            .machine("mps2-an385")
+            .monitor(qemu_config::Monitor::null)
+            .kernel(format!("{target_dir}/example.elf"))
+            .serial(qemu_config::Serial::null)
+            .no_graphic(true)
+            .snapshot(true)
+            .drives([qemu_config::Drive::builder()
+                .interface(qemu_config::DriveInterface::none)
```

Comment:
Stupid questions without understanding the details:
Do you actually need to specify `none` here? could `Option::None` make sense?

Reply:
> Do you actually need to specify none here?

I'd say yes because, in Qemu, `none` is not the default interface

> And, if so, could Option::None make sense?

mm I don't think so